### ISSUE TITLE
docs-bug(cdk/layout): LayoutModule is not needed

### ISSUE THREAD

**Issue by gultyaev** [angular/components#24248]
### Documentation Feedback

In the docs it's mentioned that we need to import LayoutModule in order to use BreakpointObserver and MediaMatcher.
In the meantime from the source code it looks like the module is not needed for any purposes.
![image](https://user-images.githubusercontent.com/22374911/150513070-a7c88a8e-e97d-481d-a4c6-e7d16492b7f2.png)
![image](https://user-images.githubusercontent.com/22374911/150513187-e52ca46b-96f5-4061-932a-ef8a100c3013.png)
![image](https://user-images.githubusercontent.com/22374911/150513301-e8f7fc77-ca5a-4230-b568-cdb0fba94c3d.png)

### Affected documentation page

https://material.angular.io/cdk/layout/api

**Comment by crisbeto** [comment]
I think that this is a leftover from before `providedIn: 'root'` existed.

**Comment by gultyaev** [comment]
Perhaps it's worth using "providedIn: LayoutModule" so the services could be included in lazy loaded modules only instead of the app module?

**Comment by crisbeto** [comment]
These services were always intended to be global so that the cached state can be reused as much as possible.

**Comment by gultyaev** [comment]
I get that. However if you use `providedIn: Module` the service isn't re-created. The instance will be shared and it won't appear in the initial bundle if was used in lazy loaded modules only

**Comment by crisbeto** [comment]
The same is true for `providedIn: 'root'` which the services are using.

**Comment by gultyaev** [comment]
Not quite. If I use it in lazy loaded module only it will increase my initial bundle size.

**Comment by gultyaev** [comment]
However, webpack is capable of making a chunk shared across other lazy loaded chunks.
